src/NamespaceDependencyChecker.php

```diff
@@ -123,32 +123,23 @@ private function analyzeFile($file)
                     if (preg_match('/\Wuse\s+\\\?'.str_replace('\\', '\\\\', $usedEntityNamespaceT).';/', $fileContent)) {
                         continue 2;
                     }
-                    if (preg_match('/\Wuse\s+\\\?[a-zA-Z0-9_\\\]+\sas\s'.$aliasName.';/', $fileContent)) {
-                        continue 2;
```

Comment:
No need to run this in this loop as nothing that changes in the iteration has an effect on the regex.